File: hw/aymacana/u1/hw07ChickenFarmSimulator2/ChickenFarmSimulator2PY/egg.py
class Egg:
    def __init__(self, size):
        self.size = size  # 'S', 'M' o 'L'

    def __str__(self):
        return f"Egg{{size = {self.size}}}"

    def get_size(self):
        return self.size

    def set_size(self, size):
        self.size = size

File: hw/aymacana/u1/hw14Chickenfarm/Java/src/model/Egg.java
package model;
/**
 *
 * @author Mateo Aymacaña
 */
public class Egg {
    private char size;

    @Override
    public String toString(){
        return "Egg{" + "size=" + size + '}';
    }
    
    public Egg(char size){
        this.size = size;
    }
    
    public char getSize(){
        return size;
    }
    
    public void setSize(char size){
        this.size = size;
    }
}

